eng/pipelines/mirror-repos.yml

```diff
@@ -53,6 +53,18 @@ jobs:
               Azure/azure-sdk-for-python-pr:
               azure-sdk/azure-sdk-for-python:
 
+          Azure/azure-sdk-for-ios:
+            Branch: master
+            TargetRepos: 
+              Azure/azure-sdk-for-ios-pr:
+              azure-sdk/azure-sdk-for-ios:
+
+          Azure/azure-sdk-for-android:
+            Branch: master
```

Comment:
You can mirror the dev branch by changing master to dev here.

Reply:
If we need both master and dev then we have to workaround a current limitation by adding the repo again below in the second template inclusion see python release/v3 branch below as an example.